main.go

```diff
@@ -26,6 +26,7 @@ var (
 	pager       bool
 	style       string
 	width       uint
+	walk		bool
```

Comment:
To avoid such situations, we agreed to the convention of using `gomt` or `goimports` to automatically format source code before committing anything. I can only recommend enabling that in your development environment of choice.

Reply:
I have not worked with golang before. Do you have a development env setup that I could checkout? Thank you 🙇

Reply:
I'm sure I can point you in the right direction! Which editor do you use?